app/src/main/java/com/kamron/pogoiv/updater/AppUpdateLoader.java

```diff
@@ -27,10 +26,19 @@ public void run() {
         try {
```

Comment:
I *guess* so, because `assets` is an array and I hope the the API is sensible enough. I hope there are docs somewhere...
The "nice" thing is, even I can't test this on this project without triggering autoupdates. `/latest` also skips prereleases which could be good. (Or maybe we should do the check ourselves and allow users to run on beta, but all of that requires more testing to get right).